app/views/teachers/_form.html.erb

```diff
@@ -49,6 +49,18 @@
       </div>
     </div>
 
+    <div class='form-group row'>
+      <div class='col-12'>
+        <%= f.label :education_level, "What's your education level target?", class: "label-required" %>
+        <%= f.select(
+          :education_level,
+          options_for_select(Teacher.education_levels, teacher.education_level_before_type_cast),
+          { include_blank: "Select an option" },
+          { class: 'form-control', required: true }
+        ) %>
+      </div>
+    </div>
+
```

Comment:
does this work with edit?
you can login as admin and see if you can edit a teacher's info.

Reply:
It should work now. It wasn't before because we forgot to sanitize the params before updating and editing. Thanks for catching this.